playbook/app/pb_kits/playbook/pb_background/docs/_background_image.html.erb

```diff
@@ -1,5 +1,5 @@
-<%= pb_rails("background", props: { image_url: "https://images.unsplash.com/photo-1528459801416-a9e53bbf4e17?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1000&q=80", padding: "lg" }) do %>
-  <%= pb_rails("flex", props: { horizontal: "center", orientation: "column" }) do %>
+<%= pb_rails("background", props: { classname: "background-image", image_url: "https://images.unsplash.com/photo-1528459801416-a9e53bbf4e17?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1000&q=80", padding: "lg" }) do %>
```

Comment:
Does this still work correctly if the user passes in a custom `classname`?

Reply:
@jasperfurniss Hey jasper, I just tested this out with another prop in the background kit and assigned it a unique class, i can confirm that setting up a custom `classname` will still allow the transition prop to work.